docs/data/charts/label/PieLabel.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { PieChart } from '@mui/x-charts/PieChart';
+
+export default function PieLabel() {
+  return (
+    <PieChart
+      {...props}
+      series={[
+        {
+          data: [
+            { id: 0, value: 10, label: (v) => `${v}+A` },
+            { id: 1, value: 15, label: (v) => `${v}+B` },
+            { id: 2, value: 20, label: (v) => `${v}+C` },
```

Comment:
WDYT about replacing `v` with `location` to make it more clear? 🤔 
```suggestion
            { id: 0, value: 10, label: (location) => `${location}+A` },
            { id: 1, value: 15, label: (location) => `${location}+B` },
            { id: 2, value: 20, label: (location) => `${location}+C` },
```